python/graphstorm/gconstruct/utils.py

```diff
@@ -997,12 +1005,36 @@ def partition_graph(g, node_data, edge_data, graph_name, num_partitions, output_
                  num_test != g.number_of_nodes(ntype)):
             balance_arr -= 1
         balance_ntypes[ntype] = balance_arr
-    mapping = \
-        dgl.distributed.partition_graph(g, graph_name, num_partitions, output_dir,
-                                        part_method=part_method,
-                                        balance_ntypes=balance_ntypes,
-                                        balance_edges=True,
-                                        return_mapping=save_mapping)
+
+
+    # Call DGL's partition graph, using the appropriate arg list
+    # for each version
+    dgl_version_str = importlib.metadata.version("dgl")
+    dgl_version = version.parse(dgl_version_str)
+    part_position_args = [
+        g, graph_name, num_partitions, output_dir,
+    ]
+    part_kwargs = {
+        'part_method': part_method,
+        'balance_ntypes': balance_ntypes,
+        'balance_edges': True,
+        'return_mapping': save_mapping,
+    }
```

Comment:
We use this way to call `dgl.distributed.partition_graph` otherwise pylint errors out because the `partition_graph` function signature is different between DGL 1.x and 2.x